src/helm/benchmark/scenarios/code_scenario.py

```diff
@@ -48,6 +48,7 @@ def check(candidate):
 Each instance has 1) a problem description with examples (as what you get in
 programming competitions), 2) coding solutions, 3) test cases.
 """
+
```

Comment:
Do you know why these extra line changes are coming up?

Reply:
Maybe because I was using a newer `black`, it added/deleted these lines. Do I need to revert these by commiting another change?

Reply:
yes that would be great!